verify_filenames.py

```diff
@@ -63,21 +63,28 @@
 
     for f in folder.iterdir():
 
-       # Test if there are any nested folders
+        # Test if there are any nested folders
 
-       if f.is_dir():
+        if f.is_dir():
            print(f"\tFound nested directory: {folder.stem}/{f.stem}")
            status = False
-       
```

Comment:
Diff looks strange here. Has intellij messed with indentation?

Reply:
I was editing on github, so there was probably some extra whitespace I left in by accident

Reply:
nevermind, it was the opposite - there was some trailing whitespace I removed